cmd/issuer-rest/go.mod

```diff
@@ -79,6 +79,7 @@ require (
 	github.com/sirupsen/logrus v1.8.1 // indirect
 	github.com/spaolacci/murmur3 v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/square/go-jose v2.4.1+incompatible // indirect
```

Comment:
Why do we need this? I already see we have go-jose v3. Is `go mod tidy` adding this as I don't see any other go mod file changes in this PR?